### PR TITLE
Activity new #29

### PR DESCRIPTION
**Pull request by chasticot**
Hello @GaetanRole , @Dine16 ,

I fixed the errors on the html and scss.

New modifications : (on new and edit form)

-Added a jquery script for show and hide address and date inputs / and achievement
-Updated the html for the address and date / and achievement (twig condition)
-Updated the fixtures (link for socialLink)
-Added Assert Not Blank for achievement

### CODE REVIEW

File: app/Resources/views/activity/edit.html.twig [hunk 1]
```diff
@@ -1,21 +1,175 @@
...
+    <main id="activity" class="light-blue-subtitle-text">
+        <section>
+            <div class="container size-align">
+                <div class="row boxing-canvas mt-50">
+                    <!--1st part of the edit form-->
+                    <!--Form Edit-->
+                    {{ form_start(edit_form) }}
+                    <div class="col-sm-10 col-sm-offset-1">
+                        <div class="col-sm-12">
+                            <h3 class="h5 degradeblue text-center mt-10">Modifier votre activité</h3>
```
Comment (GaetanRole) [inline app/Resources/views/activity/edit.html.twig]: classe h5 pour une balise h3 ?

File: app/Resources/views/activity/edit.html.twig [hunk 2]
```diff
@@ -1,21 +1,175 @@
...
+                                    {{ form_widget(edit_form.description, {'attr': {'class': "form-control input-field", 'placeholder': "Maximum 250 caractères"}}) }}
+                                    {{ form_errors(edit_form.description) }}
+                                </div>
+                            </div>
+                        </div>
+                        <!--End Activity's Description-->
+
+                        <div class="row">
+                            <div class="col-sm-12 text-center">
+                                <br>
```
Comment (GaetanRole) [inline app/Resources/views/activity/edit.html.twig]: <br> ? Utiliser une classe mt ?

File: app/Resources/views/activity/new.html.twig [hunk 3]
```diff
@@ -1,16 +1,143 @@
...
+    <main id="activity" class="light-blue-subtitle-text">
+        <section>
+            <div class="container size-align">
+                <div class="row boxing-canvas mt-50">
+                    <!--1st part of the new form-->
+                    <!--New form-->
+                    {{ form_start(form) }}
+                    <div class="col-sm-10 col-sm-offset-1">
+                        <div class="col-sm-12">
+                            <h3 class="h5 degradeblue text-center mt-10">Créer votre activité</h3>
```
Comment (GaetanRole) [inline app/Resources/views/activity/new.html.twig]: balise h3 mais classe h5

File: assets/js/input_display.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        }if($(this).val() === "Activité de streaming"){
+            // Security client view, input is required
+            $(".activityInfo").prop('required',true);
+            // Set n/a value when you want to create an activity
+            $(".activityInfo").val("n/a");
+            $(".activityAchievement").val("n/a");
+            // Hide input field
+            $(".activityInfo").hide();
+            $(".activityAchievement").hide();
+        }if($(this).val() === ""){
```
Comment (GaetanRole) [inline assets/js/input_display.js]: espace entre les if ?

File: app/Resources/views/activity/edit.html.twig (hunk 2 above)
Comment (chasticot) [reply app/Resources/views/activity/edit.html.twig]: Ok....

**Pull request closed without merging**